packages/workspace/src/builders/run-commands/run-commands.impl.spec.ts

```diff
@@ -82,24 +86,29 @@ describe('Command Runner Builder', () => {
 
     it('should run commands in parallel', async () => {
       const f = fileSync().name;
-      const run = await architect.scheduleBuilder(
+      const exec = spyOn(require('child_process'), 'exec').and.callThrough();
+      const scheduleRun = await architect.scheduleBuilder(
         '@nrwl/workspace:run-commands',
         {
           commands: [
             {
-              command: `sleep 0.2 && echo 1 >> ${f}`
+              command: `echo 1 >> ${f}`
             },
             {
-              command: `sleep 0.1 && echo 2 >> ${f}`
+              command: `echo 2 >> ${f}`
             }
           ],
           parallel: true
         }
       );
+      const processesCreated = exec.calls.count();
```

Comment:
```suggestion
      await Promise.resolve();
      const processesCreated = exec.calls.count();
```

Reply:
Is this comment in the right place? :)

Reply:
I added your suggestion to the serial runner tests 👍